### PR TITLE
Login signup

### PR DESCRIPTION
**Pull request by vishalg8454**
This PR adds Login and SignUp facility to the Video Library app.

The ReduxState slices added are-
- `authSlice` at src/store/authSlice.js`

The routes added are
- `login` at src/pages/loginpage
- `signup` at src/pages/signup-page

Would love to hear your feedback and suggestions.

[live preview](https://deploy-preview-4--videostock.netlify.app/)
image

image

### CODE REVIEW

File: src/store/authSlice.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        console.log(action.payload.message);
+        toast(action.payload.message);
+      });
+  },
+});
+
+export default authSlice.reducer;
+export const { logout } = authSlice.actions;
+
+export const loginUser = createAsyncThunk(
```
Comment (Developers-codz) [inline src/store/authSlice.js:70]: Appreciation: that you used createAsyncThunk

**Review by Developers-codz — APPROVED** [review]
Code is well structured found nothing to suggest you can merge
Small observation in UI although there is horizontal scroll fix that everything else is perfect

File: src/store/authSlice.js (hunk 1 above)
Comment (vishalg8454) [reply src/store/authSlice.js:70]: Thank you

**Review by arpit01923 — APPROVED** [review]
- code looks clean
- good to merge

**Pull request merged**